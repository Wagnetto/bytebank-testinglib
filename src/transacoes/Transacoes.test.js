import { render } from "@testing-library/react";
import React from "react";

import Transacoes from "./Transacoes";

describe("Transaction List", () => {
  it("list snapshot should keep the same", async () => {
    const listaDeTransacoes = [
        {id: 1, data:"08/09/2020", tipo:"deposito", valor:"20.00"},
        {id: 2, data:"09/09/2020", tipo:"deposito", valor:"20.00"},
        {id: 3, data:"10/09/2020", tipo:"saque", valor:"20.00"}
    ]
    const { container } = render(
      <Transacoes transacoes={listaDeTransacoes} />
    );

    expect(container.firstChild).toMatchSnapshot();
  });
});