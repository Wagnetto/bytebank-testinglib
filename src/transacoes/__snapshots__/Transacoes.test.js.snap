// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Transaction List list snapshot should keep the same 1`] = `
<div
  data-testid="transacoes"
>
  <div
    class="transacao-container"
  >
    <p>
      08/09/2020
    </p>
    <p />
    <p>
      R$ 
      20.00
    </p>
  </div>
  <div
    class="transacao-container"
  >
    <p>
      09/09/2020
    </p>
    <p />
    <p>
      R$ 
      20.00
    </p>
  </div>
  <div
    class="transacao-container"
  >
    <p>
      10/09/2020
    </p>
    <p />
    <p>
      R$ 
      20.00
    </p>
  </div>
</div>
`;
